src/Moq/Protected/ProtectedMock.cs

```diff
@@ -154,18 +154,20 @@ public ISetupSequentialResult<TResult> SetupSequence<TResult>(string methodOrPro
 
 		#region Verify
 
-		public void Verify(string methodName, Times times, object[] args)
+		public void Verify(string methodName, Times times, object[] args) { Verify(methodName, times, false, args); }
+		public void Verify(string methodName, Times times, bool exactParameterMatch, object[] args)
 		{
 			Guard.NotNullOrEmpty(methodName, nameof(methodName));
 
-			var method = GetMethod(methodName, args);
+			var method = GetMethod(methodName, exactParameterMatch, args);
 			ThrowIfMemberMissing(methodName, method);
 			ThrowIfPublicMethod(method, typeof(T).Name);
 
 			Mock.VerifyVoid(mock, GetMethodCall(method, args), times, null);
 		}
 
-		public void Verify<TResult>(string methodName, Times times, object[] args)
+		public void Verify<TResult>(string methodName, Times times, object[] args) { Verify<TResult>(methodName, times, false, args); }
```

Comment:
Please reformat (as per the above review comment) for consistency. :)

Reply:
Done